jujugui/static/gui/src/app/views/utils.js

```diff
@@ -1607,25 +1607,12 @@ YUI.add('juju-view-utils', function(Y) {
       app, app.createSocketURL.bind(app, app.get('socketTemplate')),
       app.switchEnv.bind(app), app.env, model.uuid, [model], model.name,
       env => {
-        utils._detachOnLoginHandler();
```

Comment:
Drive-by to remove the handler that was required before we changed to the switchEnv callback.